src/modules/powerrename/dll/dllmain.cpp

```diff
@@ -160,12 +161,14 @@ class PowerRenameModule : public PowertoyModuleIface
 private:
     // Enabled by default
     bool m_enabled = true;
+    wchar_t* app_name;
 
 public:
     // Return the display name of the powertoy, this will be cached
     virtual PCWSTR get_name() override
     {
-        return L"PowerRename";
+        app_name = GET_RES_STRING_WCHAR(IDS_POWERRENAME);

```

Comment:
We should set it only once

Reply:
As I changed the implementation to use a wstring instead of a wchar_t* and am no longer allocating any memory using new or malloc, I don't think there would be an issue. In my understanding I cannot see another way of obtaining the string from the resource table instead of using hardcoded strings. Please let me know if you have another approach in mind.

Reply:
Moved it to the constructor.